rel/reg1.vars.config

```diff
@@ -0,0 +1,37 @@
+%% This node is for global distribution testing.
+%% reg is short for region.
+%% "localhost" host should be defined.
+{hosts, "[ \"reg1\", \"localhost\" ]"}.
```

Comment:
`"reg1"` domain not needed

Reply:
done.

Reply:
It's not funny. Every host in GD setup needs both global domain and local domain.